### PR TITLE
Update to dutch translations

### PR DESCRIPTION
**Pull request by zegert**
Some translations were not translated yet. I have added them and can be contacted if you want more English-Dutch translations.

# Description

Fixes #(issue)
Wrong Dutch(Nederlandse) translations.

## Type of change
Changed some translations, thats really it.

Please delete options that are not relevant.

- Translation update

## Checklist

- [x ] My code follows the style guidelines of this project
- [x ] I ran ESLint and other linters for modified files
- [x ] I have performed a self-review of my own code and test it
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here and it will be uploaded automatically.

### CODE REVIEW

File: src/languages/nl-NL.js [hunk 1]
```diff
@@ -138,48 +138,48 @@ export default {
...
-    "Avg. Ping": "Avg. Ping",
-    "Avg. Response": "Avg. Response",
+    Purple: "Paars",
+    Pink: "Roze",
+    "Search...": "Zoeken...",
+    "Avg. Ping": "Gemiddelde Ping",
+    "Avg. Response": "Gemiddelde Response",
     "Entry Page": "Entry Page",
-    statusPageNothing: "Nothing here, please add a group or a monitor.",
+    statusPageNothing: "Nothing here, please add a group or a monitor. Niets hier, voeg een groep of monitor toe.",
```
Comment (deefdragon) [inline src/languages/nl-NL.js:173]: Left the English in on this one.

**Review by deefdragon — CHANGES_REQUESTED** [review]

**Comment by Saibamen** [comment]
Please run `npm run update-language-files` to sync `nl-NL.js` with latest `en.js`

File: src/languages/nl-NL.js [hunk 2]
```diff
@@ -138,48 +138,48 @@ export default {
...
     Inactive: "Inactief",
-    "Also apply to existing monitors": "Also apply to existing monitors",
+    "Also apply to existing monitors": "Voeg ook toe aan bestaande monitors",
     Token: "Token",
     "Show URI": "Toon URI",
     "Clear all statistics": "Wis alle statistieken",
-    retryCheckEverySecond: "Retry every {0} seconds.",
-    importHandleDescription: "Choose 'Skip existing' if you want to skip every monitor or notification with the same name. 'Overwrite' will delete every existing monitor and notification.",
-    confirmImportMsg: "Are you sure to import the backup? Please make sure you've selected the right import option.",
+    retryCheckEverySecond: "Probeer elke {0} seconds.",
```
Comment (koen20) [inline src/languages/nl-NL.js:145]:
```suggestion
    retryCheckEverySecond: "Probeer elke {0} seconden.",
```

**Review by koen20 — CHANGES_REQUESTED** [review]

File: src/languages/nl-NL.js (hunk 1 above)
Comment (ZegertBoele) [reply src/languages/nl-NL.js:173]: I will fix this, Thanks for looking into it.

**Pull request merged**